metricflow/dataflow/builder/dataflow_plan_builder.py

```diff
@@ -720,12 +720,9 @@ def _get_semantic_model_names_for_measures(self, measures: Sequence[MeasureSpec]
 
         This is a temporary method for use in assertion boundaries while we implement support for multiple semantic models
         """
-        semantic_model_names: Set[str] = set()
-        for measure in measures:
-            semantic_model_names = semantic_model_names.union(
-                {d.name for d in self._semantic_model_lookup.get_semantic_models_for_measure(measure.reference)}
-            )
-        return semantic_model_names
+        return {
+            self._semantic_model_lookup.get_semantic_model_for_measure(measure.reference).name for measure in measures
+        }
```

Comment:
Do you mind deleting this function and moving this one-liner over to where we call it? I think the docstring is super duper outdated and your change here removed all of the complexity from getting this set.

Reply:
Will do!